app/src/main/java/org/jellyfin/androidtv/util/CoroutineUtils.kt

```diff
@@ -0,0 +1,8 @@
+@file:JvmName("CoroutineUtils")
+package org.jellyfin.androidtv.util
+
+import kotlinx.coroutines.CoroutineScope
+
+fun <T> runBlocking(block: suspend CoroutineScope.() -> T) = kotlinx.coroutines.runBlocking {
```

Comment:
the linter says this needs to be annotated as null or non-null

Reply:
Fixed, had to use `<T: Any>` so it can never be null.